src/store/lokapi.ts

```diff
@@ -33,21 +33,31 @@ export var moduleLokAPI = {
         commit('auth_error')
         throw err
       }
-      await dispatch('setBackends')
-
-      commit("setThisWeekTransactions")
       commit('auth_success')
     },
     async resetTRS({commit} :any) {
-      await commit("setThisWeekTransactions")
+      commit("setTransactions", [])
```

Comment:
This doesn't do the same thing at all, if this change was expected, I don't understand it yet.

Reply:
It's part of a bigger change. I should not have done this. Just discard it.